### PR TITLE
Update email layouts and copy

### PR DESCRIPTION
**Pull request by Guntrisoft**
Updating the format of emails and copy within them based on feedback from the Content team

**Appointment confirmation**
pension wise appointment request
http localhost 3001 rails mailers appointments customer part text 2fplain

**Appointment reminder**
pension wise appointment request 1
http localhost 3001 rails mailers appointments reminder part text 2fplain

**Booking request (Booking manager)**
pension wise appointment request 2
http localhost 3001 rails mailers booking_requests booking_manager part text 2fplain

**Booking request confirmation**
pension wise appointment request 3
http localhost 3001 rails mailers booking_requests customer part text 2fplain

### CODE REVIEW

File: app/views/appointments/customer.html.erb [hunk 1]
```diff
@@ -1,51 +1,39 @@
-<p style="color: #0B0C0C;font-family: Helvetica, Arial, sans-serif;margin: 15px
-0;font-size: 16px;line-height: 1.315789474;">Dear <%= @appointment.name %>,</p>
+<%= h1 do %>
+  <% if @appointment.updated? %>
+    Your appointment details were updated.
+  <% else %>
+    We've booked your appointment.
```
Comment (benbarnett) [inline app/views/appointments/customer.html.erb]: http://smartquotesforsmartpeople.com/

File: app/views/appointments/customer.html.erb [hunk 2]
```diff
@@ -1,51 +1,27 @@
...
+    Thank you for booking a Pension Wise appointment.
   <% end %>
 </p>
 
-<h3 style="color: #0B0C0C;font-family: Helvetica, Arial, sans-serif">Preparing for your appointment</h3>
-
-<p style="color: #0B0C0C;font-family: Helvetica, Arial, sans-serif;margin: 15px
-0;font-size: 16px;line-height: 1.315789474;">To make your appointment as useful
-as possible, it would be helpful to know the following information:</p>
+<%= render 'shared/email/html/appointment_details' %>
```
Comment (benlovell) [inline app/views/appointments/customer.html.erb]: I thinking splitting the HTML / text parts into separate folders is not necessary. If you keep them in the same folder with the same filenames and suffixed with the correct type (text.erb or html.erb) they should be correctly inferred.

File: app/views/booking_requests/customer.html.erb [hunk 3]
```diff
@@ -1,44 +1,67 @@
...
-<p style="color: #0B0C0C;font-family: Helvetica, Arial, sans-serif;margin: 15px
-0;font-size: 16px;line-height: 1.315789474;">Your reference number<br> <strong
-class="emphasize" style="font-size: 24px;"><%=
-  @booking_request.reference %></strong></p>
-
-<p style="color: #0B0C0C;font-family: Helvetica, Arial, sans-serif;margin: 15px
-0;font-size: 16px;line-height: 1.315789474;">To cancel or change your
-appointment, or if any of the information above isn't correct, call
- <%= @booking_location.online_booking_twilio_number %> or reply to this email.</p>
+<p style="color: #0B0C0C;font-family: Helvetica, Arial, sans-serif;margin: 15px 0;font-size: 16px;line-height: 1.315789474;">
```
Comment (benlovell) [inline app/views/booking_requests/customer.html.erb]: Maybe we ought to render this `p` in a helper so we're not repeating the long `style` attribute everywhere? Would cut down on a lot of noise here.

File: app/views/shared/email/html/_appointment_details.html.erb [hunk 4]
```diff
@@ -0,0 +1,50 @@
+<p style="color: #0B0C0C;font-family: Helvetica, Arial, sans-serif;margin: 15px 0;font-size: 16px;line-height: 1.315789474;">
```
Comment (benlovell) [inline app/views/shared/email/html/_appointment_details.html.erb]: Same here with the helper thang.

File: app/views/shared/email/html/_preparing_for_appointment.html.erb [hunk 5]
```diff
@@ -0,0 +1,44 @@
+<h2 style="color: #0B0C0C !important;font-family: Helvetica, Arial, sans-serif;margin: 15px 0;font-weight: 700;font-size: 18px;line-height: 1.111111111;padding: 15px 0;">
+  Preparing for your appointment
+</h2>
+
+<p style="color: #0B0C0C;font-family: Helvetica, Arial, sans-serif;margin: 15px 0;font-size: 16px;line-height: 1.315789474;">To make the most of your appointment it would be helpful if you know:</p>
```
Comment (benlovell) [inline app/views/shared/email/html/_preparing_for_appointment.html.erb]: Same same.

File: app/views/booking_requests/customer.html.erb (hunk 3 above)
Comment (Guntrisoft) [inline app/views/booking_requests/customer.html.erb]: That's what I was trying to achieve through my last version of this PR - using helpers for each tag to cut down on repetition - maybe i'll have a go at a half way house with just the paragraphs.

Comment (benlovell) [inline app/views/booking_requests/customer.html.erb]: Yeah but without inventing a new language.

File: app/views/appointments/customer.html.erb (hunk 2 above)
Comment (Guntrisoft) [inline app/views/appointments/customer.html.erb]: fair - done

File: app/views/booking_requests/customer.html.erb (hunk 3 above)
Comment (Guntrisoft) [inline app/views/booking_requests/customer.html.erb]: done

**Comment by Guntrisoft** [comment]
[Review and amend customer emails](https://trello.com/c/94zwkatg/12-review-and-amend-customer-emails)

**Pull request merged**